test/unit/SourceLocator/Type/InvalidDirectorySourceLocatorTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace BetterReflectionTest\SourceLocator\Type;
+
+use BetterReflection\SourceLocator\Exception\InvalidDirectory;
+use BetterReflection\SourceLocator\Type\DirectorySourceLocator;
+
+/**
+ * @covers \BetterReflection\SourceLocator\Type\DirectorySourceLocator
```

Comment:
This shouldn't be tested here. You should have one test class for the exception, one for the locator